LL/de_rigaib_muman_iar_cretim_inso.ttl

```diff
@@ -0,0 +1,580 @@
+
+<#FergusScandail>
+    a irishTitles:Rí ;
+    foaf:name "Fergus Scandail" ;
+    rel:childOf <#CrimthannCrom> ;
+    irishTitles:reignLength 12 ;
+    irishTitles:reignEndedBy "ec"@sga ;
+    irishTitles:succeededBy <#Garban> ;
+    irishTitles:succeededBy <#Garbith>.
+
+<#Garban>
+    a irishTitles:Rí ;
+    foaf:name "Garban" ;
+    irishTitles:reignLength 7 ;
+    irishTitles:reignEndedBy "marb"@sga ;
+    irishTitles:succeededBy <#Amalgaid>.
+
+<#Garbith>
+    a irishTitles:Rí ;
+    foaf:name "Garbith" ;
+    irishTitles:reignLength 7 ;
+    irishTitles:reignEndedBy "marb"@sga ;
+    irishTitles:succeededBy <#Amalgaid>.
+
+<#Amalgaid>
+    a irishTitles:Rí ;
+    foaf:name "Amalgaid" ;
+    irishTitles:reignLength 10 ;
+    irishTitles:reignEndedBy "marb"@sga ;
+    irishTitles:succeededBy <#AedBennain>.
+
+<#AedBennain>
+    a irishTitles:Rí ;
+    foaf:name "Aed Bennain" ;
+    irishTitles:reignLength 9 ;
+    owl:sameAs <http://example.com/LL/eoganachta_casil#AedaBennain>;
+    irishTitles:reignEndedBy "ec"@sga ;
+    irishTitles:succeededBy <#Feidlimid>.
+
+<#Feidlimid>
+    a irishTitles:Rí ;
+    foaf:name "Feidlimid" ;
+    rel:childOf <#Carpri>;
+    irishTitles:succeededBy <#Fingin>.
+
+<#Carpri>
+    a foaf:Person ;
+    foaf:name "Carpri" .
+    
+<#Fingin>
+    a irishTitles:Rí ;
+    foaf:name "Fingin" ;
+    rel:childOf <#Aeda>;
+    owl:sameAs <http://example.com/LL/eoganachta_casil.ttl#Fhingin>;
+    irishTitles:reignLength 23 ;
+    irishTitles:succeededBy <#Cathal>.
+
+<#Aeda>
+    owl:sameAs <http://example.com/LL/eoganachta_casil.ttl#AedaDuib>.
+
+<#Cathal>
+    a irishTitles:Rí ;
+    foaf:name "Cathal" ;
+    rel:childOf <#Aeda-1675c29e>;
+    owl:sameAs <http://example.com/LL/eoganachta_casil.ttl#Cathail-dd8d5bfe>;
+    irishTitles:reignLength 20 ;
+    irishTitles:reignEndedBy "ec"@sga ;
+    irishTitles:succeededBy <#FalbeFland>.
+
+<#Aeda-1675c29e>
+    owl:sameAs <http://example.com/LL/eoganachta_casil.ttl#AedaFlaindCathrach>.
+
+<#FalbeFland>
+    a irishTitles:Rí ;
+    foaf:name "Falbe Fland" ;
+    rel:childOf <#Aeda-70251d9b>;
+    irishTitles:reignLength 20 ;
+    irishTitles:reignEndedBy "ec"@sga ;
+    irishTitles:succeededBy <#Cuan>.
+    #sameAs http://example.com/LL/eoganachta_casil.ttl#Falbe-d0114e86 ? - CY
+
+<#Aeda-70251d9b>
+    a foaf:Person ;
+    foaf:name "Aeda" .
+    
+<#Cuan>
+    a irishTitles:Rí ;
+    foaf:name "Cuan" ;
+    rel:childOf <Amalgaid-eae7a30c> ;
+    owl:sameAs <http://example.com/LL/h_n_enna.ttl#Cuain>;
+    irishTitles:reignLength 10 ;    
+    irishTitles:reignEndedBy "ec"@sga, "marbad"@sga ;
+    irishTitles:succeededBy <#Moenach>.
+
+<Amalgaid-eae7a30c>
+    owl:sameAs <http://example.com/LL/h_n_enna.ttl#Amalgada>.
+
+<#Moenach>
+    a irishTitles:Rí ;
+    foaf:name "Moenach" ;
+    rel:childOf <#Fingin-337a7443>;
+    irishTitles:reignLength 12 ;
+    irishTitles:reignEndedBy "ec"@sga ;
+    irishTitles:succeededBy <#CucenMathair>.
+
+<#Fingin-337a7443>
```

Comment:
This is possibly <#Fingin>, above, a previous king. The timing is just about right for the next generation to be taking the kingship.